woocommerce/woocommerce
# Add phone number to my account shipping address.

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR adds the phone field to Shipping Address edit form under My Account, on par with the phone field presented on Billing Address form.
<img width="600" alt="image" src="https://github.com/woocommerce/woocommerce/assets/17236129/b9407d44-d9cd-473a-ade5-9190dee42727">

Closes #45717



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Edit the Checkout page, and within the Block click the Address section. On the sidebar enable the phone number field:
<img width="316" alt="image" src="https://github.com/woocommerce/woocommerce/assets/17236129/a678c163-9cff-4bd2-8538-cea0759f604a">

2. On a logged-in WC installation, go to My Account > Addresses > Shipping Address edit
3. Verify that the Phone field is display, and saving a phone number persists
4. Verify that under My Account > Addresses Billing Address edit the Phone field also exists and saving a phone number persists
5. Verify that it is possible to add different phone numbers to each of the address types
6. Create an order and supply different phone numbers for each address
7. Log in as admin, and open the order under WooCommerce > Orders
8. Verify on the order details that the submitted phone numbers are correctly displayed



### Changelog entry




<details>

#### Significance




#### Type




#### Message 
Added phone number field to Shipping Address form under My Account
#### Comment 

</details>
